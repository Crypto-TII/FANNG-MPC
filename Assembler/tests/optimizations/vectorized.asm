vaddm 100, s42,c42,c142
addci s0, s43, 9
