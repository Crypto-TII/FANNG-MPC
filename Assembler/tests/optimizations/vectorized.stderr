warning: Write to register is never read
 --> tests/optimizations/vectorized.asm:2:1
  |
2 | addci s0, s43, 9
  | ---------------- register 0 written to but will never be read from again
  |
warning: Write to register is never read
 --> tests/optimizations/vectorized.asm:1:1
  |
1 | vaddm 100, s42,c42,c142
  | ----------------------- register 42 written to but will never be read from again
  |
error: read from uninitialized register
 --> tests/optimizations/vectorized.asm:1:1
  |
1 | vaddm 100, s42,c42,c142
  | ^^^^^^^^^^^^^^^^^^^^^^^ register 42 was never written to
  |
error: read from uninitialized register
 --> tests/optimizations/vectorized.asm:1:1
  |
1 | vaddm 100, s42,c42,c142
  | ^^^^^^^^^^^^^^^^^^^^^^^ register 142 was never written to
  |